src/modules/75-cf/components/FlagArchiving/useArchiveFlagDialog.tsx

```diff
@@ -46,36 +107,65 @@ const useArchiveFlagDialog = ({
     }
   }
 
+  const getDialogContent = () => {
```

Comment:
Currently this gets re-evaluated and contents rerendered on each rerender of the the dialog. A better idea would be to put it behind a `useMemo` to stabilise the references.
```suggestion
  const dialogContent = useMemo(() => {
```